interactor-impl/src/test/java/com/wednesday/template/interactor/weather/search/SearchCityInteractorImplTest.kt

```diff
@@ -83,7 +84,7 @@ class SearchCityInteractorImplTest : InteractorTest() {
                 val result = awaitItem()
 
                 // Then
-                assertEquals(expected = uiList, actual = result)
+                assertTrue(result is UIResult.Success)
```

Comment:
Let's keep both the asserts. `result` could be an instance of `Success` but the internal data might be different.

Reply:
The problem is that result is `UiResult` and the expected stuff is `UIResult.Success(data)`
That's why i've just checked for success
We can cast and check the data inside the Success subclass if required but I don't think that's necessary